torchao/experimental/kernels/mps/src/OperationUtils.h

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) Meta Platforms, Inc. and affiliates.
+// All rights reserved.
+//
+// This source code is licensed under the license found in the
+// LICENSE file in the root directory of this source tree.
+
+#pragma once
+
+#include <iostream>
+
+static void fail(const std::string& str) {
+  std::cerr << str << std::endl;
+  abort();
```

Comment:
This looks more C++ to me.. (I.e. callee can catch the error if need be)
```suggestion
  throw std::runtime_error(str);
```